controllers/module_reconciler.go

```diff
@@ -118,6 +124,18 @@ func (r *ModuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	r.setKMMOMetrics(ctx)
 
+	if mod.Spec.ModuleLoader.ServiceAccountName == "" {
+		logger.Info("Handle default driver container ServiceAccount")
+		if err := r.handleDriverContainerServiceAccount(ctx, mod); err != nil {
+			return res, fmt.Errorf("could not handle default driver container ServiceAccount: %w", err)
+		}
+
+		logger.Info("Handle default driver container RoleBinding")
+		if err := r.handleDriverContainerRoleBinding(ctx, mod); err != nil {
+			return res, fmt.Errorf("could not handle default driver container RoleBinding: %w", err)
```

Comment:
```suggestion
		logger.Info("Handle default module loader RoleBinding")
		if err := r.handleModuleLoaderRoleBinding(ctx, mod); err != nil {
			return res, fmt.Errorf("could not handle default module loader RoleBinding: %w", err)
```

Reply:
Same as in https://github.com/rh-ecosystem-edge/kernel-module-management/pull/53#discussion_r981121561.